### PR TITLE
Remove Old: JanDeDobbeleer.OhMyPosh version 4.2.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
null

### CODE REVIEW

File: manifests/v/valinet/ExplorerPatcher/Prerelease/22000.348.40.11_94db02b/valinet.ExplorerPatcher.Prerelease.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: valinet.ExplorerPatcher.Prerelease
+PackageVersion: 22000.348.40.11_94db02b
```
Comment (ItzLevvie) [inline manifests/v/valinet/ExplorerPatcher/Prerelease/22000.348.40.11_94db02b/valinet.ExplorerPatcher.Prerelease.installer.yaml:5]:
```suggestion
PackageVersion: 22000.348.40.11
```

File: manifests/v/valinet/ExplorerPatcher/Prerelease/22000.348.40.11_94db02b/valinet.ExplorerPatcher.Prerelease.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: valinet.ExplorerPatcher.Prerelease
+PackageVersion: 22000.348.40.11_94db02b
```
Comment (ItzLevvie) [inline manifests/v/valinet/ExplorerPatcher/Prerelease/22000.348.40.11_94db02b/valinet.ExplorerPatcher.Prerelease.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 22000.348.40.11
```

File: manifests/v/valinet/ExplorerPatcher/Prerelease/22000.348.40.11_94db02b/valinet.ExplorerPatcher.Prerelease.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: valinet.ExplorerPatcher.Prerelease
+PackageVersion: 22000.348.40.11_94db02b
```
Comment (ItzLevvie) [inline manifests/v/valinet/ExplorerPatcher/Prerelease/22000.348.40.11_94db02b/valinet.ExplorerPatcher.Prerelease.yaml:5]:
```suggestion
PackageVersion: 22000.348.40.11
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by OfficialEsco** [comment]
Duplicate of https://github.com/microsoft/winget-pkgs/pull/37704

**Comment by Trenly** [comment]
@vedantmgoyal Can you check on what needs to be added for the versionMethod?

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**